packages/babel-plugin-transform-es2015-parameters/src/default.js

```diff
@@ -1,137 +0,0 @@
-import getFunctionArity from "babel-helper-get-function-arity";
```

Comment:
I should be able to look at the transform logic for this in the next day or two of someone else doesn't get to it. My main comment from skimming this is that I'm not sure I think the `es2015-parameters` transform should leave in the `...` parameters. Leaving in the `...` is fine for `object-rest-spread` because it is only dealing with object-rest, but until now the `parameter` transform` has also removed all usage of rest params, leaving the params as standard ES5, and this PR would change that. It might be better for this PR to also convert the rest params to `arguments` usage.

That said, alternatively we could consider repurposing this plugin to leave in the rest params, but that should be an explicit decision we make. As it is, I'd probably have expected one PR to update the implementation to your new logic without breaking changes, with potentially a second PR to change to using rest params.